__tests__/app.test.js

```diff
@@ -88,6 +88,80 @@ describe("app", () => {
               });
           });
         });
+        describe("PATCH: /api/reviews/:review_id", () => {
+          test("200: responds with updated review object when vote increased", () => {
+            const incReviewVotes = {
+              inc_votes: 1,
+            };
+
+            return request(app)
+              .patch("/api/reviews/1")
+              .send(incReviewVotes)
+              .expect(200)
+              .then(({ body: { review } }) => {
+                expect(review).toEqual({
+                  review_id: 1,
+                  title: "Agricola",
+                  designer: "Uwe Rosenberg",
+                  owner: "mallionaire",
+                  review_img_url:
+                    "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+                  review_body: "Farmyard fun!",
+                  category: "euro game",
+                  created_at: "2021-01-18T10:00:20.514Z",
+                  votes: 2,
+                });
+              });
+          });
+          test("200: responds with updated review object when vote decreased", () => {
+            const decReviewVotes = {
+              inc_votes: -3,
+            };
+
+            return request(app)
+              .patch("/api/reviews/2")
+              .send(decReviewVotes)
+              .expect(200)
+              .then(({ body: { review } }) => {
+                expect(review).toEqual({
+                  review_id: 2,
+                  title: "Jenga",
+                  designer: "Leslie Scott",
+                  owner: "philippaclaire9",
+                  review_img_url:
+                    "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+                  review_body: "Fiddly fun for all the family",
+                  category: "dexterity",
+                  created_at: "2021-01-18T10:01:41.251Z",
+                  votes: 2,
+                });
+              });
+          });
+          test("400: responds with error when body is missing required fields", () => {
+            const badBody = {};
+
+            return request(app)
+              .patch("/api/reviews/1")
+              .send(badBody)
+              .expect(400)
+              .then(({ body: { message } }) => {
+                expect(message).toBe("Bad request");
+              });
+          });
+          test("400: responds with error when passed key value of wrong type", () => {
+            const badBody = {
+              inc_votes: "one vote",
+            };
+
+            return request(app)
+              .patch("/api/reviews/1")
+              .send(badBody)
+              .expect(400)
+              .then(({ body: { message } }) => {
+                expect(message).toBe("Bad request");
+              });
+          });
```

Comment:
Please also test for when a vote is attempted on an id that doesn't exist, or one that's of the wrong type.